spanner/tests/system/test_system.py

```diff
@@ -728,6 +728,28 @@ def _unit_of_work(transaction, test):
 
         return session, committed
 
+    def test_read_with_multiple_keys_index(self):
+        row_count = 10
+        ddl_create = ['CREATE INDEX name ON contacts(first_name, last_name)']
+        columns = self.COLUMNS[1], self.COLUMNS[2]
+        try:
+            operation = self._db.update_ddl(ddl_create)
+            operation.result(120)
+            session, committed = self._set_up_table(row_count, db=self._db)
+            self.to_delete.append(session)
+            expected = [[row[1], row[2]] for row in self._row_data(row_count)]
+            keyset = [(row[0], row[1]) for row in expected]
```

Comment:
This is still redundant:  `keyset` is exactly equal to `expected`.

Reply:
There was a time where I believe there was a mismatch between tuple and lists on the output side, so I think that was what I changed.  But you are right and I now removed the redundancy.